obal/data/tasks/download_and_extract_source.yml

```diff
@@ -99,6 +99,14 @@
         path: "{{ spec_file_path }}"
         regexp: '^(?P<prefix>Source0:\s+)http.*/(?P<filename>.*)$'
         replace: '\g<prefix>http://sat-r220-02.lab.eng.rdu2.redhat.com/pulp/isos/Sat6-CI-Source_Files-Satellite_6_3_Source_Files/\g<filename>'
+      when: product_name != 'rhui'
+
+    - name: 'Ensure Source0 specfile points to dogfood'
+      replace:
+        path: "{{ spec_file_path }}"
+        regexp: '^(?P<prefix>Source0:\s+)http.*/(?P<filename>.*)$'
+        replace: '\g<prefix>http://sat-r220-02.lab.eng.rdu2.redhat.com/pulp/isos/e3fb5ed0-8126-4b7a-b380-d101c97814b3/\g<filename>'
```

Comment:
Noticing why this change is needed, I'd prefer to get this URLs out of this upstream project and into their package_manifests as configuration pleeeaassseee.